.env.example

```diff
@@ -36,13 +32,12 @@ AFM_CONNECTION_TIMEOUT=10000
 AFM_KEY=key
 
 ECOMMERCE_STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;QueueEndpoint=http://storage:10001/devstoreaccount1"
-TRANSACTION_AUTHORIZATION_REQUESTED_EVENT_QUEUE_NAME=transactionauthrequestedtevents
-TRANSACTION_AUTHORIZATION_REQUESTED_EVENT_QUEUE_VISIBILITY_TIMEOUT=60
 
 TRANSACTION_EXPIRATION_QUEUE_NAME=pagopa-ecommerce-transactions-expiration-queue
-TRANSACTION_CLOSEPAYMENT_RETRY_QUEUE_NAME=pagopa-ecommerce-transactions-closepayment-retry-queue
+TRANSACTION_CLOSE_PAYMENT_RETRY_QUEUE_NAME=pagopa-ecommerce-transactions-close-payment-retry-queue
```

Comment:
maybe it should also be updated ecommerce-local with these changes absolutely correct.
What do you think? 🙂

Reply:
Done into this [PR](https://github.com/pagopa/pagopa-ecommerce-local/pull/62) (as linked in description section)

Reply:
Sorry 🙂